modules/logging-loki-restart-hardening.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-restart-hardening_{context}"]
+= LokiStack behavior during cluster restarts
+
+In logging version 5.8 and newer versions, when an {product-title} cluster is restarted, Loki log ingestion and queries continue to operate within the available CPU and memory resources available for the node. This means that there is no downtime for the LokiStack during {product-title} cluster updates. This behavior is achieved by using `PodDisruptionBudget` resources. The Loki Operator provisions `PodDisruptionBudget` resources for Loki, which determine the minimum number of pods that must be available per component to ensure normal operations under certain conditions.
```

Comment:
Just a minor suggestion from my side to have 'LokiStack ingestion and query path' instead of 'Loki log ingestion and queries'

Reply:
Done, thank you!